desk/src/pages/ticket/TicketCustomerTemplateFields.vue

```diff
@@ -6,6 +6,22 @@
         {{ transformStatus(ticket.data.status) }}
       </span>
     </div>
+    <div class="space-y-1.5">
+      <span class="block text-sm text-gray-700"> Response By </span>
+      <span class="block break-words text-base font-medium text-gray-900">
+        <Tooltip :text="dayjs(ticket.data.response_by).long()">
+          {{ dayjs(ticket.data.response_by).fromNow() }}
+        </Tooltip>
+      </span>
+    </div>
+    <div class="space-y-1.5">
+      <span class="block text-sm text-gray-700"> Resolution By </span>
+      <span class="block break-words text-base font-medium text-gray-900">
+        <Tooltip :text="dayjs(ticket.data.resolution_by).long()">
```

Comment:
We are using hacky workaround of long SLA time as "no sla" so that won't look so good on user portal. Rest looks fine.

Reply:
Let's just add a Fake SLA as "No SLA" and random dates for now as we discussed. Tried to refactoring this and the only time it worked was when I removed the whole code 😆 

This would need a deeper refactor